be/src/docker-compose.override.yml

```diff
@@ -276,6 +277,9 @@ services:
       - OpenTelemetryTracingCollectorUrl=http://otel-collector:4317
       - OpenTelemetryMetricsCollectorUrl=http://otel-collector:4317
       - OpenTelemetryLogsCollectorUrl=http://otel-collector:4317
+      - OrganisationId=09affcc9-1665-45d6-919f-3d2026106ba1
```

Comment:
**suggestion:** Add variables to pipelines. I will show you how to do it on next sprint daily :)